metpy/calc/tests/test_calc_tools.py

```diff
@@ -77,6 +77,17 @@ def test_find_intersections_invalid_direction():
         find_intersections(x, y1, y2, direction='increaing')
 
 
+def test_find_intersections_units():
+    """Test handling of units when logarithmic interpolation is called."""
```

Comment:
I like the test, so let's keep it. But now I just noticed the `@wraps` decorator on `find_intersections`, so I think that's managing units for you already--that's why coverage still shows the unit-handling code as not running. You may be able to eliminate it altogether then.

Reply:
Yup that works - hadn't noticed that and need to get more comfortable dealing with decorators. Tests were still green after removing those lines, so hopefully everything stays green this time.